File: Ember.Handlebars.ExampleWebAPI/Scripts/app/models/show.js
/// <reference path="../../ember-1.0.0-rc.2.min.js" />
/// <reference path="../../ember-data.js" />
/// <reference path="../app.js" />

App.Show = DS.Model.extend({
    title: DS.attr('string'),
    network: DS.attr('string'),
    category: DS.attr('string'),
    year: DS.attr('number')    
});

File: Ember.Handlebars.ExampleWebAPI/Models/Show.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace Ember.Handlebars.ExampleWebAPI.Models {
    
    public class Show {

        [Key]
        public int ShowId { get; set; }
        [StringLength(50)]
        public string Title { get; set; }
        [StringLength( 50 )]
        public string Category { get; set; }
        [StringLength( 50 )]
        public string Network { get; set; }
        public int Year { get; set; }

    }
}